channels/channels_fsm.go

```diff
@@ -95,9 +95,9 @@ var ChannelEvents = fsm.Events{
 		chst.AddLog("data transfer receive error: %s", chst.Message)
 		return nil
 	}),
-	fsm.Event(datatransfer.RequestTimedOut).FromAny().ToNoChange().Action(func(chst *internal.ChannelState, err error) error {
+	fsm.Event(datatransfer.RequestCancelled).FromAny().ToNoChange().Action(func(chst *internal.ChannelState, err error) error {
```

Comment:
Since we are changing this state transition and RequestTimedOut is abandoned now, should we migrate all RequestTimedOut to RequestCancelled, so that they're not left in limbo? Or are these terminal states?

Alternatively, we could rename RequestTimedOut to RequestCancelled?

Reply:
RequestTimedOut & RequestCancelled are events, not states, so I don't think they will get recorded anywhere. I don't think we need to migrate anything.